Windows.md

```diff
@@ -0,0 +1,82 @@
+<!---
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License. See accompanying LICENSE file.
+-->
+
+# Building parquet-cpp on Windows
+
+## Fast setup of building requirements with conda and conda-forge
+
+To not waste a lot of time to setup the thirdparty build dependecies for
```

Comment:
Some might disagree that it's a waste of time, but it's indeed _convenient_, so "A convenient and tested way to set up thirdparty dependencies is to use the conda package manager."